_providers/fastmail.md

```diff
@@ -33,3 +33,7 @@ at "Settings / Import & Setup / Setup Guides / Thunderbird / Next":
 Then, click "Next" again and then "Done".
 
 Afterwards you can use Delta Chat as usual.
+
+### Custom domains
```

Comment:
comparable texts (gmail, web.de) use "headline-level-2" here, so i would go for that as well here.

```suggestion
## Custom domains
```